src/script/modules/settings-menu/components/settings/UnsaveableSnaps.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import useSettingState from '../../../../hooks/useSettingState';
+import Switch from '../Switch';
+
+const NAME = 'Send Unsaveable';
```

Comment:
I think renaming this to `Unsaveable Messages / Snaps`, would give more clarity

"Send Unaveable" doesn't really tell you anything